README.md

```diff
@@ -45,15 +45,18 @@ As go plugins can be finicky to correctly compile and install, you may want to c
 # We use go modules for everything.
 > export GO111MODULE=on
 
+# (Optionally) clean go cache
+go clean -modcache; go clean -cache
+
 # Clone kubo.
 > git clone https://github.com/ipfs/kubo
 > cd kubo
 
 # Pull in the datastore plugin (you can specify a version other than latest if you'd like).
-> go get github.com/ipfs/go-ds-s3/plugin@latest
+> go get github.com/ipfs/go-ds-s3/go-ds-s3-plugin@latest
```

Comment:
```suggestion
> go get github.com/ipfs/go-ds-s3@latest
```

Reply:
Applied in [`80ac067`](https://github.com/ipfs/go-ds-s3/pull/275/commits/80ac06715a1e0d5d890f1c8215d85b946bfc2f62)